src/systems/explicit_system.C

```diff
@@ -54,22 +57,8 @@ void ExplicitSystem::clear ()
   // Clear the parent data
   Parent::clear();
 
-  // NULL-out the vector.  Note that
-  // System::clear() actually deleted it.
-  rhs = libmesh_nullptr;
-}
-
-
-
-void ExplicitSystem::init_data ()
-{
-  // Add the system RHS.
-  // (We must do this before initializing the System data,
-  //  then we lose the opportunity to add vectors).
+  // Restore us to a "basic" state
   this->add_system_rhs ();
```

Comment:
Me too, but this appears necessary for restoring. It also follows @roystgnr 's coding in `TransientSystem::clear`.